src/pages/UserDashboard/Donations/Table.tsx

```diff
@@ -79,6 +79,9 @@ export default function Table({
             USD Value
           </HeaderButton>
           {status === "intent" ? <></> : <>TX Hash</>}
+          {status === "pending" && (
```

Comment:
@jp-mariano  @SovereignAndrey, perhaps col name should be more generic like `Actions` 
<img width="1162" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/c24d78bd-05c8-4a33-9e6f-a0c834d8dbd6">

as other donations may not be `bank`, in this case `Action` could just be `- - - `

Reply:
Addressed in commit https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/3107/commits/d375820a1fa43d14f9bf704dfbb6782271fde22d. Also updated PR's description copy.